app/views/purchases/new.html.erb

```diff
@@ -0,0 +1,38 @@
+<div class="navigation">
+  <ul>
+    <li><%= link_to image_tag('backarrow.png', alt: 'Back'), category_path(params[:category_id]), class: 'nav-link' %></li>
+    <li><h2>Add a new Purchase</h2></li>
+    <li><%= link_to 'Sign Out', destroy_user_session_path, data: { turbo_method: :delete }, class: 'nav-link' %></li>
+  </ul>
+</div>
+
+<div class="form-class">
+  <%= form_with(model: [@category, @purchase], url: category_purchases_path(@category), method: :post, id: 'purchase-form') do |form| %>
+    <div class="form-field">
+      <%= form.label :name, 'Name for Transaction', class: 'form-label' %>
+      <%= form.text_field :name, class: 'form-control', placeholder: 'Enter Name for Transaction' %>
+    </div>
+    
+    <div class="form-field">
+      <%= form.label :amount, 'Amount for Transaction', class: 'form-label' %>
+      <%= form.text_field :amount, class: 'form-control', placeholder: 'Enter Amount for Transaction' %>
+    </div>
+    
+  <!-- Hidden field for primary category -->
+  <%= form.hidden_field :primary_category_id, value: @category.id %>
+
+  <!-- Additional Categories -->
+  <div class="form-field">
+    <p>Additional Categories:</p>
+    <% @categories.each do |category| %>
+      <% next if category == @category %>
+      <div class="checkbox-field">
+        <%= check_box_tag 'additional_category_ids[]', category.id, false, id: "additional_category_ids_#{category.id}", class: 'form-checkbox' %>
+        <%= label_tag "additional_category_ids_#{category.id}", category.name, class: 'checkbox-label' %>
+      </div>
+    <% end %>
+  </div>
+
+  <%= form.submit 'Create Purchase', class: 'btn btn-primary' %>
+  <% end %>
+</div>
```

Comment:
- [ ] Kindly design the **_form page_** for adding a new **_transaction_** to look identical to that of the design layout provided for forms. In the design layout, a form page for registering a user was provided. Kindly use the design layout for other form pages in your application. Designing a page based on a wireframe or sample layout ensures consistent and accurate implementation of the intended user experience, reducing ambiguity and misinterpretation. 🖱️ 🖥️ 


| Your Design  | Behance Design |
| ------------- | ------------- |
| ![image](https://github.com/misalazeem/Budget-App/assets/77106962/238306cc-1000-489d-8b00-fc6aab8e6308) | ![image](https://github.com/yegonkimutai/budget-app/assets/77106962/1167219c-4a3b-48e4-b19d-22ebb21c5edd) |

#### Hint: You don't need the form labels. 👍 😎